samples/src/JustSaying.Sample.Restaurant.KitchenConsole/BusService.cs

```diff
@@ -24,7 +25,7 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
         {
             _logger.LogInformation("Kitchen subscriber running");
 
-            _bus.Start(stoppingToken);
+            _bus.StartAsync(stoppingToken);
```

Comment:
Awaiting this would pause until the bus is stopped. Startup here is synchronous until the run tasks are collected and added together in a `WhenAll`, which is returned and awaitable.

Granted there's not a lot of use cases for awaiting this outside of tests.

Reply:
If it's intentional, then I'd just make it explicit with a discard:

```csharp
_ = _bus.StartAsync(stoppingToken);
```